### PR TITLE
Article siblings

### PR DESCRIPTION
**Pull request by davidpmccormick**
## Who is this for?
People who want an appropriate piece of related content on a story

## What is it doing for them?
Reinstating the behaviour we wanted, but must have been without for some time

If we're reading a serial (i.e. a series with a schedule), the related content at the end of it will be the next part (or the first part if the one we're reading is the final part).

### CODE REVIEW

File: content/webapp/pages/article.tsx [hunk 1]
```diff
@@ -257,12 +257,22 @@ const ArticlePage: FC<Props> = ({ article }) => {
...
+            i => i.partNumber === 1
+          );
+          const firstArticleTitle = firstArticleFromSchedule?.title;
+          const firstAticle = articles.find(i => i.title === firstArticleTitle);
+
+          const nextArticleFromSchedule = series.schedule.find(
+            i => i.partNumber === positionInSerial + 1
+          );
+          const nextArticleTitle = nextArticleFromSchedule?.title;
+          const nextAticle = articles.find(i => i.title === nextArticleTitle);
```
Comment (jamieparkinson) [inline content/webapp/pages/article.tsx:270]:
```suggestion
          const firstArticle = articles.find(i => i.title === firstArticleTitle);

          const nextArticleFromSchedule = series.schedule.find(
            i => i.partNumber === positionInSerial + 1
          );
          const nextArticleTitle = nextArticleFromSchedule?.title;
          const nextArticle = articles.find(i => i.title === nextArticleTitle);
```

File: content/webapp/pages/article.tsx [hunk 2]
```diff
@@ -257,12 +257,22 @@ const ArticlePage: FC<Props> = ({ article }) => {
...
+
           const nextUp =
             positionInSerial === series.schedule.length
-              ? series.items[0]
-              : series.items[positionInSerial]
-              ? series.items[positionInSerial]
-              : null;
+              ? firstAticle
+              : nextAticle || null;
```
Comment (jamieparkinson) [inline content/webapp/pages/article.tsx:275]:
```suggestion
              ? firstArticle
              : nextArticle || null;
```

**Review by jamieparkinson — APPROVED** [review]
Do we have no choice other than to compare titles (ie do we have IDs or something)?

**Comment by davidpmccormick** [comment]
> Do we have no choice other than to compare titles (ie do we have IDs or something)?

Sadly no

**Pull request merged**